geniza/corpus/management/commands/import_data.py

```diff
@@ -127,7 +128,8 @@ def source_setup(self):
 
         # delete source records created on a previous run
         Creator.objects.all().delete()
-        Source.objects.all().delete()  # should cascade to footnotes
+        Source.objects.all().delete()
+        Footnote.objects.all().delete()
```

Comment:
Did this because deletions didn't actually seem to cascade (or at least PKs kept incrementing...is that normal?)